operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java

```diff
@@ -33,9 +33,8 @@
 
   String ENCODED_CREDENTIALS = "encodedCredentials";
   String DOMAIN_VALIDATION_ERRORS = "domainValidationErrors";
-
+  //  String UNRECOVERABLE_INTROSPECT_ERRORS = "unrecoverableIntrospectErrors";
```

Comment:
shall we remove the added code that is commented out?

Reply:
Yes, it should be and shouldn't be there